components/api/src/main/java/com/hotels/styx/api/HttpResponse.java

```diff
@@ -571,7 +571,7 @@ Builder ensureContentLengthIsValid() {
 
         private static boolean isInteger(String contentLength) {
             try {
-                parseInt(contentLength);
+                parseLong(contentLength);
```

Comment:
do you think we should validate it is a positive integer / greater than 0?

Reply:
Done